x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/flyout/index.tsx

```diff
@@ -101,7 +102,7 @@ export const useToGetInternalFlyout = () => {
           isIsolateActionSuccessBannerVisible={isIsolateActionSuccessBannerVisible}
           isHostIsolationPanelOpen={isHostIsolationPanelOpen}
           loading={isLoading || loading}
-          rawEventData={rawEventData}
+          rawEventData={rawEventData as AlertRawEventData}
```

Comment:
why this cast? Also this flyout is used with regular events as well, so I think this cast is dangerous as what it's cast too won't exist for non-alerts

Reply:
Previously `rawEventData` was of 'object | undefined' type - I am not sure what this data contains, but I thought that my interface would be better than `object`. Another reason for doing this was: I had no other idea ;P Do you think it would be better to if I name it differently?

Reply:
Ok I thought it through, and I removed the cast, and casted it down below in the place I actually need it. 
Thank you for catching this! :)